supplementary_style_guide/style_guidelines/formatting.adoc

```diff
@@ -83,6 +83,11 @@ For example, you might combine existing product name attributes to create compou
 :name-spring-reactive: {name-runtime-spring-boot} with {name-runtime-vertx} reactive components
 ----
 
+[[titles-and-headings]]
+== Titles and headings
+
+Write titles and headings in sentence case, for example, _Composing a customized RHEL system image_. Do not use title case.
```

Comment:
"Titles" could also be interpreted to mean titles for figures, tables, etc. Can you update this to specifically call out that we mean guide titles? The example you give didn't strike me as a guide title even though I just realized upon poking around that it was. Probably could add an example section (in response to SSG survey about more examples). Maybe something like:

```suggestion
Write all headings, including guide titles, in sentence case. Do not use title case.

.Examples

* _Composing a customized RHEL system image_
* <one or two other examples>
```

Reply:
Made your suggested changes.